dev/automated_tests/pubspec.yaml

```diff
@@ -13,10 +13,10 @@ dependencies:
   integration_test:
     sdk: flutter
   platform: 3.1.0
-  test: 1.24.3
+  test: 1.24.4
```

Comment:
It looks like bumping this is causing version solving on the plugins repo to fail:

```
Because test >=1.16.0-nullsafety.19 <1.16.6 depends on test_api 0.2.19 and test >=1.16.6 <1.17.10 depends on analyzer ^1.0.0, test >=1.16.0-nullsafety.19 <1.17.10 requires test_api 0.2.19 or analyzer ^1.0.0.
And because test >=1.17.10 <1.20.0 depends on analyzer >=1.0.0 <3.0.0 and test >=1.20.0 <1.21.2 depends on test_api 0.4.9, test >=1.16.0-nullsafety.19 <1.21.2 requires test_api 0.2.19 or 0.4.9 or analyzer >=1.0.0 <3.0.0.
And because test >=1.21.2 <1.21.3 depends on test_api 0.4.10 and test >=1.21.3 <1.21.4 depends on test_api 0.4.11, test >=1.16.0-nullsafety.19 <1.21.4 requires test_api 0.2.19 or 0.4.9 or 0.4.10 or 0.4.11 or analyzer >=1.0.0 <3.0.0.
And because test >=1.21.4 <1.21.5 depends on test_api 0.4.12 and test >=1.21.5 <1.21.6 depends on test_api 0.4.13, test >=1.16.0-nullsafety.19 <1.21.6 requires test_api 0.2.19 or 0.4.9 or 0.4.10 or 0.4.11 or 0.4.12 or 0.4.13 or analyzer >=1.0.0 <3.0.0.
And because test >=1.21.6 <1.21.7 depends on test_api 0.4.14 and test >=1.21.7 <1.22.0 depends on test_api 0.4.15, test >=1.16.0-nullsafety.19 <1.22.0 requires test_api 0.2.19 or 0.4.9 or 0.4.10 or 0.4.11 or 0.4.12 or 0.4.13 or 0.4.14 or 0.4.15 or analyzer >=1.0.0 <3.0.0.
And because test >=1.22.0 <1.22.1 depends on test_api 0.4.16 and test >=1.22.1 <1.22.2 depends on test_api 0.4.17, test >=1.16.0-nullsafety.19 <1.22.2 requires test_api 0.2.19 or 0.4.9 or 0.4.10 or 0.4.11 or 0.4.12 or 0.4.13 or 0.4.14 or 0.4.15 or 0.4.16 or 0.4.17 or analyzer >=1.0.0 <3.0.0.
And because test >=1.22.2 <1.24.0 depends on test_api 0.4.18 and test >=1.24.0 <1.24.3 depends on matcher >=0.12.15 <0.12.16, test >=1.16.0-nullsafety.19 <1.24.3 requires test_api 0.2.19 or 0.4.9 or 0.4.10 or 0.4.11 or 0.4.12 or 0.4.13 or 0.4.14 or 0.4.15 or 0.4.16 or 0.4.17 or 0.4.18 or analyzer >=1.0.0 <3.0.0 or matcher >=0.12.15 <0.12.16.
And because test >=1.24.3 <1.24.4 depends on test_api 0.6.0 and every version of flutter_test from sdk depends on matcher 0.12.16, if test >=1.16.0-nullsafety.19 <1.24.4 and flutter_test from sdk then test_api 0.2.19 or 0.4.9 or 0.4.10 or 0.4.11 or 0.4.12 or 0.4.13 or 0.4.14 or 0.4.15 or 0.4.16 or 0.4.17 or 0.4.18 or 0.6.0 or analyzer >=1.0.0 <3.0.0.
And because test >=1.24.4 depends on analyzer >=5.12.0 <7.0.0 and pigeon >=3.2.7 <7.0.5 depends on analyzer ^4.4.0, if pigeon >=3.2.7 <7.0.5 and flutter_test from sdk and test >=1.16.0-nullsafety.19 then test_api 0.2.19 or 0.4.9 or 0.4.10 or 0.4.11 or 0.4.12 or 0.4.13 or 0.4.14 or 0.4.15 or 0.4.16 or 0.4.17 or 0.4.18 or 0.6.0.
And because url_launcher_windows depends on flutter_test from sdk which depends on test_api 0.6.1, pigeon >=3.2.7 <7.0.5 is incompatible with test >=1.16.0-nullsafety.19.
So, because url_launcher_windows depends on both pigeon ^5.0.1 and test ^1.16.3, version solving failed.
```

Reply:
@aam in order to roll vm_snapshot_analysis, maybe we should pin package:test with a `TODO(fujino): roll flutter/packages deps`

Reply:
I had a https://github.com/flutter/packages/pull/4383 in mind as a fix for this.

Reply:
Oh nice, thanks!